### DISCUSSION TITLE
profile condition youtube uploader (display-tags) (yt-dlp)

### DISCUSSION THREAD

**Discussion by geextahslex** [mpv-player/mpv#14485] [Q&A]
Hi, I want to make a profile with the condition of the youtube uploader, in this example it's "Kacey". How can I set this up?

Thank you :)

![kacey](https://github.com/mpv-player/mpv/assets/139760266/f5ffd4f5-c5d6-4ebb-8752-a021f04f8061)

**Comment by llyyr** [comment]
https://mpv.io/manual/master/#command-interface-metadata

**Reply by geextahslex** [reply]
I don't understand. Can you be more specific?
